x-pack/plugins/lens/mappings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "lens": {
+    "properties": {
+      "title": {
+        "type": "text"
+      },
+      "visualizationType": {
+        "type": "keyword"
+      },
+      "datasourceType": {
+        "type": "keyword"
+      },
+      "lensState": {
```

Comment:
When we started out with this branch we decided to keep the name "lens" out of the code as much as possible to make a conversion to a Kibana "standard" easier later on. It makes total sense to call the saved object type itself lens as it's a separate thing for now, but we could also call this key just `state`, right? Then we have one place less to worry about later